website/google.erb

```diff
@@ -108,6 +108,9 @@
       <li<%= sidebar_current("docs-google-datasource-iam-policy") %>>
```

Comment:
This is missing both new storage object access control resources (google_storage_object_access_control, google_storage_default_object_access_control), and likely other new resources as well.

Not sure if every resource is relevant, but they were in https://github.com/terraform-providers/terraform-provider-google-beta/commit/bed58517b90ab72c16f45d3a628c621e94a56b5f